assets/base.css

```diff
@@ -572,6 +572,11 @@ table:not([class]) th {
   text-align: center;
 }
 
+.justify {
+  text-align: justify;
+  text-align-last: justify;
```

Comment:
it feels so weird that such a basic text formatting thing doesn't work on a browser... 🤷 
![](https://screenshot.click/30-01-0sd93-jy0jd.png)

There is a work around mentioned there but it sounds wacky 🤔

Reply:
I am curious to why we are adding a `Justify` option?

I don't believe justifying text works well on the web at all where we can't control the amount of words, the language and the width of devices the content will be read. Merchants won't be able to control the additional spacing between the words that this will create. But I would love to learn more of use cases where this might be successful though!

We also need to consider that if we are adding this in one place as an option, we would want to offer it elsewhere and would require more in depth explorations and testing.